signbank/dictionary/adminviews.py

```diff
@@ -269,6 +269,14 @@ def get_queryset(self):
             val = get['location']
             qs = qs.filter(location=val)
 
+        if 'example_search' in get and get['example_search'] != '':
+            val = get['example_search']
+            val = '[' + val + ']'
```

Comment:
Can you add a comment here to explain for future devs why we do this? We should have enough content in Slack to copy-paste some stuff here.